magento2-vulnerable-extensions.csv

```diff
@@ -16,3 +16,7 @@ Magestore_Storelocator,1.0.2,,,https://blog.magestore.com/store-locator-extensio
 Magestore_Storepickup,1.2.1,,,https://blog.magestore.com/store-locator-extension-patch/
 Mirasvit_Blog,1.0.21,,,https://github.com/mirasvit/module-blog
 Mirasvit_Giftr,1.2.23,,https://mirasvit.com/docs/module-gift-registry/current/changelog,https://mirasvit.com/magento-2-extensions/gift-registry.html
+lof_formbuilder,,,https://anothernetsecblog.com/magento-2-extension-security/
+blueformbuilder,,,https://anothernetsecblog.com/magento-2-extension-security/
+vladimirpopov_webforms,2.9.18,,https://anothernetsecblog.com/magento-2-extension-security/
+lof_chatsystem,,,https://anothernetsecblog.com/magento-2-extension-security/
```

Comment:
Are these really the module names (from module.xml). Typically they use initial caps (like all the others Mirasvit_Giftr). Also, blueformbuilder doesn't seem right?

Reply:
Mate you're 100% correct, my bad on that ill update this now.